recipes/opencv/4.x/conanfile.py

```diff
@@ -163,6 +163,7 @@ def _patch_opencv(self):
             tools.replace_in_file(find_openexr, "SET(OPENEXR_LIBSEARCH_SUFFIXES Win32/Release Win32 Win32/Debug)", "")
 
         tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), "ANDROID OR NOT UNIX", "FALSE")
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), "elseif(EMSCRIPTEN)", "elseif(EMSCRIPTEN)\nelseif(QNXNTO)")
```

Comment:
I think this breaks emscripten builds.
Does the following suggestion work?
```suggestion
        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), "elseif(EMSCRIPTEN)", "elseif(EMSCRIPTEN OR QNXNTO)")
```

Reply:
In this case it does not brake it, as EMSCRIPTEN also does nothing:
https://github.com/opencv/opencv/pull/20045/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR652-R655

I can turn around the two, so that if EMSCRIPTEN would ever have system libraries linked, it would still work with that patch?
```suggestion
        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), "elseif(EMSCRIPTEN)", "elseif(QNXNTO)\nelseif(EMSCRIPTEN)")
```

Reply:
I guess your suggestion would be the safest approach.

Reply:
done